scripts/covid_hosp_prod.R

```diff
@@ -97,226 +113,266 @@ rlang::list2(
         select(-disease) %>%
         filter(geo_value %nin% insufficient_data_geos)
       most_recent_result
-    },
-    cue = tar_cue("always")
+    }
   ),
-  tar_target(
+  tar_change(
     name = nhsn_archive_data,
+    change = get_s3_object_last_modified("forecasting-team-data", "archive_timestamped.parquet"),
     command = {
       create_nhsn_data_archive(disease = "nhsn_covid")
     }
   ),
+  # TODO: tar_change maybe?
```

Comment:
we should either switch this to `tar_change` or leave it as always mode

Reply:
makes sense. i'm hoping tar_change works. i think it should, unless there's something borked with the timestamp getting code, since tar_change is just shorthand for a two-target chain.